python/nano/test/pytorch/tests/inference/test_ipex_jit_inference.py

```diff
@@ -116,7 +126,16 @@ def hello():
         new_model.hello()
         with pytest.raises(AttributeError):
             new_model.width
-        
+
+        # test ipex inplace
+        new_model = InferenceOptimizer.trace(model, use_ipex=True, inplace=True)
```

Comment:
Will `model` be changed after `inplace=True`? I am not sure will it affect the following tests?

Reply:
Good catch, although it looks don't affect below test, but I may define a new model here.